cmd/podman/spec.go

```diff
@@ -556,8 +559,9 @@ func (c *createConfig) GetContainerCreateOptions() ([]libpod.CtrCreateOption, er
 		options = append(options, libpod.WithName(c.Name))
 	}
 	// TODO parse ports into libpod format and include
-	// TODO should not happen if --net=host
-	options = append(options, libpod.WithNetNS([]ocicni.PortMapping{}))
+	if !c.NetMode.IsHost() {
```

Comment:
I think this should also be eliminated on IsNone and IsContainer?

Reply:
Should return an error if the user requested port mappings on an invalid ntwork type.

Reply:
Yeah, once port mapping is actually implemented, I would agree.